src/exo/platforms/neon.py

```diff
@@ -169,3 +192,23 @@ def neon_vfmadd_1xf32_4xf32(
 
     for i in seq(0, 4):
         dst[i] += lhs[0] * rhs[i]
+
+
+# TODO: Hack for procedure aliasing issue, can be deleted once we have
+#      better way of handling aliasing
```

Comment:
I am unsure if this is only useful as a hack due to aliasing. I could imagine this pattern showing up when you are doing more automation.